scripts/lint.py

```diff
@@ -9,10 +9,10 @@
 from __future__ import annotations
 
 import argparse
-from glob import glob
 import os
 import re
 import sys
+from glob import glob
```

Comment:
Why wasn't this caught by CI? 🤔

Reply:
Looked into it and fixed it, see https://github.com/rerun-io/rerun/pull/5995